addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -521,6 +521,14 @@ def _import_fill_invoice_form(self, journal, tree, invoice_form, qty_factor):
                 logs.append(_("Could not retrieve currency: %s. Did you enable the multicurrency option "
                               "and activate the currency ?", currency_code_node.text))
 
+        # ==== Bank Details ====
+
+        bank_detail_nodes = tree.findall('.//{*}PaymentMeans')
+        bank_details = [bank_detail_node.findtext('{*}PayeeFinancialAccount/{*}ID') for bank_detail_node in bank_detail_nodes]
```

Comment:
This should remove falsy values right, then we don't need to keep checking
```suggestion
        bank_details = [bank_detail_node.findtext('{*}PayeeFinancialAccount/{*}ID') for bank_detail_node in bank_detail_nodes if bank_detail_node]
```